VncSharp/Framebuffer.cs

```diff
@@ -79,7 +79,7 @@ public Rectangle Rectangle
 	    }
 
 	    /// <summary>
-		/// The number of Bits Per Pixel for the Framebuffer--one of 8, 24, or 32.
+		/// The number of Bits Per Pixel for the Framebuffer--one of 8, 16, or 32.
```

Comment:
What happened to 24?

Reply:
As per the rfbproto.h document from http://www.hep.phy.cam.ac.uk/vnc_docs/, there is only bitsPerPixel values of 8, 16, or 32 (extract shown below).

/*−−−−−−−−−−−−−−−−−−−−−−−−−−−−−−−−−−−−−−−−−−−−−−−−−−−−−−−−−−−−−−−−−−−−−−−−−−−−−
* Structure used to specify pixel format.
*/
typedef struct {
CARD8 bitsPerPixel; /* 8,16,32 only */

Reply:
Indeed you're right.  https://tools.ietf.org/html/rfc6143#section-7.4 also agrees with this.